src/main/java/board/Board.java

```diff
@@ -0,0 +1,65 @@
+package board;
+
+import java.util.HashMap;
+import java.util.Map;
+import piece.Piece;
+import piece.TeamType;
+
+public class Board {
+
+    private final Map<Position, Piece> map;
+
+    public Board(final Map<Position, Piece> map) {
+        this.map = new HashMap<>(map);
+    }
+
+    public boolean existPieceByPosition(final Position existPosition) {
```

Comment:
```suggestion
    public boolean existPieceByPosition(final Position position) {
```
사소하지만 position에 아직 존재하는지 안하는지 모르니 parameter명도 이렇게 수정해주면 좋습니다.

Reply:
헉 맞습니다! 세세한 리뷰 감사합니다 ☺️